jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/openshift/OpenShiftBuildServiceUtils.java

```diff
@@ -133,7 +134,7 @@ protected static BuildStrategy createBuildStrategy(
               .withNamespace(StringUtils.isEmpty(fromNamespace) ? null : fromNamespace)
             .endFrom()
             .withEnv(checkForEnv(imageConfig))
-            .withBuildArgs(Optional.ofNullable(buildConfig.getArgs()).orElse(Collections.emptyMap()).entrySet().stream()
+            .withBuildArgs(Optional.of(mergeBuildArgs(imageConfig, jKubeServiceHub.getConfiguration())).orElse(Collections.emptyMap()).entrySet().stream()
```

Comment:
as far as I remember, mergeBuildArgs is also taking prop from the local docker config which is not applicable for openshift remote docker build.

Reply:
You're right. How shall we handle this situation? Maybe add a `boolean` parameter in `mergeBuildArgs` to check whether to read local docker config or not?